ci/publish.sh

```diff
@@ -1,16 +1,29 @@
 #!/usr/bin/env bash
 
 # This is an entrypoint for CI build step, don't change it's relative path(name)
-# force Envs be set up
-set -u
 
 source $(dirname $0)/_main.sh
-local readonly profileId="-Pavito.ossrh.stagingProfileId=${$OSSRH_STAGING_ID}"
-local readonly user="-Pavito.ossrh.user=${OSSRH_USER}"
-local readonly password="-Pavito.ossrh.password=${OSSRH_PASSWORD}"
 
-local readonly CREDENTIALS = "${profileId} ${user} ${password}"
+# force Envs be set up
+set -uex
+function publish() {
+    # PGP_KEY is a large 4kb key with unescaped symbols
+    local readonly gradle_properties_path="subprojects/gradle.properties"
+    if [[ -f ${gradle_properties_path} ]]; then
+        echo "avito.pgp.key=$PGP_KEY" >> ${gradle_properties_path}
```

Comment:
So, I couldn't use `docker env-list`

Reply:
Why?

Reply:
It didn't work as expected, so I decided to solve a problem how I can